src/main/java/org/folio/service/titles/TitlesService.java

```diff
@@ -73,11 +72,15 @@ public Future<Title> getTitleById(String titleId, RequestContext requestContext)
   }
 
   public Future<Void> saveTitle(Title title, RequestContext requestContext) {
-    String targetTenantId = TenantTool.tenantId(requestContext.getHeaders());
-    return inventoryInstanceManager.createShadowInstanceIfNeeded(title.getInstanceId(), targetTenantId, requestContext)
-      .compose(shadowInstance -> {
+    return saveTitleWithInstance(title, false, requestContext).mapEmpty();
```

Comment:
here should be the same logic, as in previous create() method, firstly invoke createShadowInstanceIfNeeded - if instance has not been created because non ECS mode - create instance with inventoryInstanceManager.getOrCreateInstanceRecord